packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panels_builder.dart

```diff
@@ -31,6 +39,27 @@ class KnowledgePanelsBuilder {
           allPanels: knowledgePanels,
         ));
       }
+      if (product != null && appLocalizations != null) {
+        if (panelId == 'health_card') {
```

Comment:
Nit: Is there a constant for this string value anywhere?

Reply:
I could not find `'health_card'` anywhere else in smoothie or off-dart.